devel/openssl/Portfile

```diff
@@ -52,8 +53,8 @@ if {${os.platform} eq "darwin" && ${os.major} < 11} {
 
 # https://github.com/macports/macports-ports/commit/9ce0f93609adf300e84bdd599366a3d4ab05215d
 # https://github.com/devnexen/openssl/commit/cef404f1e7a598166cbc2fd2e0048f7e2d752ad5
```

Comment:
These two lines can probably be removed now that you're removing the patch referenced here?

Reply:
The first one seems to simply reference the previous update to the port, which seems like a strange thing to have included, anyway.  But the second one references the quasi-upstream patch, which is modified but not removed here, hence the "patch updated" comment.

Reply:
I updated it to reference the two relevant upstream commits from the actual upstream repo, plus the comment about fixing the upstream version for 10.4.

I only retested the comment-only change on one system. :-)